### ISSUE TITLE
about the 'n_embedding'

### ISSUE THREAD

**Issue by george-kris** [limaosen0/Variational-Graph-Auto-Encoders#1]
how to initialize the n_embedding from arg? thank you !

**Comment by johnqoe** [comment]
> 
> 
> how to initialize the n_embedding from arg? thank you !

Have you solved this problem? I also encountered this problem.

**Comment by george-kris** [comment]
no. but i found the original code provided by the GAE's author. https://github.com/tkipf/gae
if you want to use the code provided by limaosen0, i guess you need train the graph first, get the embedding an use it. 




------------------ 原始邮件 ------------------
发件人: "johnqoe"<notifications@github.com>;
发送时间: 2019年3月16日(星期六) 下午5:18
收件人: "limaosen0/Variational-Graph-Auto-Encoders"<Variational-Graph-Auto-Encoders@noreply.github.com>;
抄送: "木子"<1228195997@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [limaosen0/Variational-Graph-Auto-Encoders] about the'n_embedding' (#1)



 
how to initialize the n_embedding from arg? thank you !
  
Have you solved this problem? I also encountered this problem.

**Comment by johnqoe** [comment]
> 
> no. but i found the original code provided by the GAE's author. https://github.com/tkipf/gae if you want to use the code provided by limaosen0, i guess you need train the graph first, get the embedding an use it.
> […](#)
> ------------------ 原始邮件 ------------------ 发件人: "johnqoe"<notifications@github.com>; 发送时间: 2019年3月16日(星期六) 下午5:18 收件人: "limaosen0/Variational-Graph-Auto-Encoders"<Variational-Graph-Auto-Encoders@noreply.github.com>; 抄送: "木子"<1228195997@qq.com>; "Author"<author@noreply.github.com>; 主题: Re: [limaosen0/Variational-Graph-Auto-Encoders] about the'n_embedding' (#1) how to initialize the n_embedding from arg? thank you ! Have you solved this problem? I also encountered this problem. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.

Great，thank you very much.

**Comment by Billy1900** [comment]
n_embedding stands for the number of the implicit units, it should be a constant number, usually, we set 128.
